src/views/includes/jurisdiction-form.njk

```diff
@@ -0,0 +1,47 @@
+<form class="form" action={{ props.currentUrl }} method="post">
+  <input type="hidden" name="pageType" value={{ props.pageType }}>  
+ 
+  {% include "includes/csrf_token.njk" %}
+      
+  {% include "includes/errors.njk" %}
+      
+  {% include "includes/proposed-name.njk" %}  
+
+  {{ govukRadios({
+    classes: "govuk-radios",
+    name: "jurisdiction",
+    fieldset: {
+      legend: {
+        text: i18n.whatIsYourJurisdiction.whereIsJurisdiction,
+        isPageHeading: true,
+        classes: "govuk-fieldset__legend--xl"
+      }
+    },
+      hint: {
+      text: "This is where the partnership was formed. The registered office address and principal place of business will need to be in the same jurisdiction."
```

Comment:
should this hint text be in i18n translations

Reply:
Using translation json.